doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Connecting to Node-RED"
+description: ""
+---
+
+[Node-RED](https://nodered.org/) is free, JavaScript-based development tool for visual programming, developed to ease the process of wiring together hardware devices, APIs and online services.
+
+The MQTT server that is exposed by the Application Server can be connected to Node-RED. This kind of integration allows setting up a Node-RED flow that listens to events and messages sent by the end devices. Also, it is possible to send messages to the end devices.
+
+This guide contains the instructions for setting up these flows, but first make sure to properly [install the Node-RED](https://nodered.org/docs/getting-started/local).
+
+## Setup
+
+First, you need to run the Node-RED by using the following command in terminal:
+```bash
+$ node-red 
+```
+```bash
+# Tip: Node-RED runs locally on port 1880 by default, so make sure to allow the service to run on this port with:
+
+$ sudo ufw allow 1880
+```
+
+>Note: you have to [secure Node-RED](https://nodered.org/docs/user-guide/runtime/securing-node-red) if you want to connect it to MQTT Server over TLS, since its editor is not secured by the default.
```

Comment:
This isn't actually required to connect over TLS, right? Of course it's good practice to secure a Node-RED deployment, but that's out of scope for our guide about connecting it to The Things Stack.

Reply:
I actually did have problems with connecting Node-RED's MQTT input node to MQTT server over port 8883, until I followed these steps for running Node-RED over https, so I thought those were required, but I guess something else could've been the issue there